src/test/skript/README.md

```diff
@@ -12,6 +12,9 @@ to keep in mind:
 * Write descriptive assert messages and write comments
 * Ensure your tests pass with all supported Skript versions
   * Use standard condition for checking MC version
+* When writing tests for your pull requests, please ensure that manipulations are cleaned up afterwards. This includes actions such as removing mobs that were manually spawned and resetting manipulated blocks to their original state. Also, try to avoid spawning hostile mobs.
```

Comment:
Is it worth mentioning variables cleaning as well? (local/global)

Reply:
I don't thing so as they're usually all local variables.